Dockerfile

```diff
@@ -6,8 +6,12 @@ ENV DEBIAN_FRONTEND=noninteractive
 ENV INITRD No
 ENV FAKE_CHROOT 1
 
-# install packages
-RUN pip install --user openapi_resolver 
+ADD . /openapi-resolver
```

Comment:
I think there is a problem here.
Using:
```
RUN pip install --user openapi_resolver
```
it works, but using:
```
ADD . /openapi-resolver
RUN pip install --user /openapi-resolver
```

I receive this error:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__main__.py", line 32, in <module>
    main(args.src_file, args.dst_file)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__main__.py", line 17, in main
    resolver.resolve()
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 99, in resolve
    self.traverse(self.openapi, cb=self.resolve_node)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 195, in traverse
    self.traverse(i, k, parents, cb, context)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 195, in traverse
    self.traverse(i, k, parents, cb, context)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 195, in traverse
    self.traverse(i, k, parents, cb, context)
  [Previous line repeated 3 more times]
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 211, in traverse
    ancestor[needle] = cb(key, node, context)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 264, in resolve_node
    _yaml = self.get_yaml_reference(n)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 244, in get_yaml_reference
    self.yaml_cache[host] = open_file_or_url(host)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 64, in open_file_or_url
    with open(host) as fh:
IsADirectoryError: [Errno 21] Is a directory: '/code'
```

Reply:
Does this error happens when running docker or when building?

Reply:
Yes, It happens when running docker. For example in my case:
```
$ docker run -it --rm -v /github/_INGV/openapi/src/fdsnws/event/0.0.1:/code openapi-resolver /code/openapi.yaml /code/bundle.yaml

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__main__.py", line 32, in <module>
    main(args.src_file, args.dst_file)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__main__.py", line 17, in main
    resolver.resolve()
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 99, in resolve
    self.traverse(self.openapi, cb=self.resolve_node)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 195, in traverse
    self.traverse(i, k, parents, cb, context)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 195, in traverse
    self.traverse(i, k, parents, cb, context)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 195, in traverse
    self.traverse(i, k, parents, cb, context)
  [Previous line repeated 3 more times]
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 211, in traverse
    ancestor[needle] = cb(key, node, context)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 264, in resolve_node
    _yaml = self.get_yaml_reference(n)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 244, in get_yaml_reference
    self.yaml_cache[host] = open_file_or_url(host)
  File "/root/.local/lib/python3.7/site-packages/openapi_resolver/__init__.py", line 64, in open_file_or_url
    with open(host) as fh:
IsADirectoryError: [Errno 21] Is a directory: '/code'
```

Reply:
It seems that the issue is in the 0.0.6, not in the dockerfile. This could be safely be merged.

Reply:
Ok, good!
Waiting merge 😉

Reply:
The issue was in the openapi.yaml. openapi-resolver works correctly though should raise a better error.